subsys/bluetooth/rpc/Kconfig

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config BT_RPC
+	bool "Bluetooth over RPC"
+	select NRF_RPC
+	select NRF_RPC_CBOR
+	help
+	  Enables Bluetooth serialization over RPC
+
+if BT_RPC
+
+choice BT_RPC_ROLE_CHOICE
+	prompt "Bluetooth over RPC role selection"
+	default BT_RPC_HOST if SOC_NRF5340_CPUNET
+	default BT_RPC_CLIENT
+	help
+	  Select the device role for Bluetooth over RPC. The default role is
+	  a client role.
+
+config BT_RPC_CLIENT
+	bool "Bluetooth over RPC Client"
+	select BT
+	help
+	  Bluetooth Low Energy API will use nRF RPC library to access host on
+	  the remote core.
+
+config BT_RPC_HOST
+	bool "Bluetooth over RPC Host"
+	select BT
+	help
+	  Bluetooth Low Energy API will use nRF RPC library to provide API for
+	  client on the remote core.
+
+endchoice
+
+config BT_RPC_INITIALIZE_NRF_RPC
+	bool "Automatically initialize nRF RPC library"
+	default y
+	help
+	  Initialize nRF RPC library during the system startup. Disabling this
+	  option allow user to initialize it in a different way.
+
+config CBKPROXY_OUT_SLOTS
```

Comment:
I completely agree with this change. The goal is to make some RPC utility sub-module that can be used by other implementations using RPC.

Additionally, long therm plan is to use something else than tinyCBOR at the bottom of `serialize.c` module.

Those are future plans, but so far only bluetooth RPC is using the modules and they may change significantly during the development process, so that's why we put it here. This PR is only the first step of whole bluetooth API serialization. 

@joakimtoe, can we agree that those modules separation will go with (or before) the second big step, which is GATT API serialization?